TECHNICAL_DETAILS.md

```diff
@@ -1,5 +1,29 @@
 # Technical Details #
 
+## Summary of supported features & options ##
+
+| Option  | Status | Override via ASG conf |
```

Comment:
How about some more columns for having the feature exposed via Terraform or CloudFormation, since experimental features may not  be exposed from the start, but just run as command-line options during development.

And also a field for the maturity of each feature